app/src/androidTest/java/com/pajato/android/gamechat/FABTest.java

```diff
@@ -60,72 +38,27 @@ public void testFABMenuPresent() {
         onView(withId(R.id.fab_speed_dial))
                 .check(matches(isDisplayed()))
                 .perform(click());
-
-        Activity rule = mRule.getActivity();
         // Ensure that the speed dial Indicators are all visible.
-        onView(withText(rule.getString(R.string.new_tictactoe_game)))
-                .check(matches(isDisplayed()));
-        onView(withText((rule.getString(R.string.new_checkers_game))))
+        onView(withText(R.string.new_chat_select_user))
                 .check(matches(isDisplayed()));
-        onView(withText((rule.getString(R.string.new_chess_game))))
+        onView(withText(R.string.new_game_settings))
                 .check(matches(isDisplayed()));
-        onView(withText((rule.getString(R.string.new_game_advanced_settings))))
+        onView(withText(R.string.new_chat_favorite_room))
                 .check(matches(isDisplayed()));
     }
 
     /**
-     * Ensure that filling the board and then using the FAB to restart actually clears the board.
+     * Ensure that, when navigating to the game pane, the FAB disappears.
      */
     @Test
-    public void testFABNewGameTTT() {
-        // Fill the board with a non-ended state
-        onView(withTagValue(is((Object) "button00")))
-                .perform(click());
-        onView(withTagValue(is((Object) "button01")))
-                .perform(click());
-        onView(withTagValue(is((Object) "button02")))
-                .perform(click());
-        onView(withTagValue(is((Object) "button10")))
-                .perform(click());
-
-        onView(withTagValue(is((Object) "button20")))
-                .perform(click());
-        onView(withTagValue(is((Object) "button21")))
-                .perform(click());
-        onView(withTagValue(is((Object) "button22")))
-                .perform(click());
-        onView(withTagValue(is((Object) "button12")))
-                .perform(click());
-
-        // Open up the FAB menu and initiate a new game
+    public void testFABNotDisplayedGameFragment() {
```

Comment:
Conciseness is also why I include the annotation, when it is short and clear, on the same line as the construct it annotates.